python/ql/src/semmle/python/frameworks/MarkupSafe.qll

```diff
@@ -0,0 +1,139 @@
+/**
+ * Provides classes modeling security-relevant aspects of the `MarkupSafe` PyPI package.
+ * See https://markupsafe.palletsprojects.com/en/2.0.x/.
+ */
+
+private import python
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.dataflow.new.TaintTracking
+private import semmle.python.Concepts
+private import semmle.python.ApiGraphs
+
+/**
+ * Provides models for the `MarkupSafe` PyPI package.
+ * See https://markupsafe.palletsprojects.com/en/2.0.x/.
+ */
+private module MarkupSafeModel {
+  /**
+   * Provides models for the `markupsafe.Markup` class
+   *
+   * See https://markupsafe.palletsprojects.com/en/2.0.x/escaping/#markupsafe.Markup.
+   */
+  module Markup {
+    /** Gets a reference to the `markupsafe.Markup` class. */
+    API::Node classRef() {
+      result = API::moduleImport("markupsafe").getMember("Markup")
+      or
+      result = API::moduleImport("flask").getMember("Markup")
```

Comment:
```suggestion
      result = API::moduleImport(["markupsafe", "flask"]).getMember("Markup")
```

Reply:
Although you certainly could, I think having these one separate lines makes the code easier to quickly scan with your eyes.

Reply:
You are right :), thought that too with [this block](https://github.com/github/codeql/blob/c77884b8c4ef0525f8acaf8c24d53cfae06e84d5/python/ql/src/semmle/python/frameworks/MarkupSafe.qll#L102-L104).